contribute.md

```diff
@@ -89,11 +89,13 @@ Please make sure there is a [Redmine issue](/contribute.html#Bugreporting) open
 git checkout -b <branchName> # Example: git checkout -b 1656-add_TB_support
 {% endhighlight %}
 
-*  Make changes and commit. Please reference the Redmine issue this commit addresses via "refs" or "fixes" #issueid in the commit message.
+*  Make changes and commit. Please reference the Redmine issue this commit
+addresses via "Refs" or "Fixes" in the commit message. Also try to keep the
+initial commit line at maximum of 60 characters.
```

Comment:
> This duplicates and contradicts the information in the handbook pages under "commit messages", it should probably link there instead.

I haven't noticed, changed to link instead. Thanks.

-- 
Later,
 Lukas #lzap Zapletal